spec/features/admin/auth_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'Auth', type: :feature do
+  context 'as signed-out user' do
+    scenario 'user tries to access' do
+      visit admin_users_path
+      expect(page).to have_http_status(:not_found)
+    end
+  end
+
+  context 'as user without admin permissions' do
+    let(:user) { create(:user, admin: false) }
+
+    scenario 'trying to access' do
+      visit admin_users_path(as: user)
+      expect(page).to have_http_status(:not_found)
```

Comment:
I think this is even better because it doesn't tell if there is an admin page. (Although any attacker could tell by the source code).